assets/app.js

```diff
@@ -859,11 +859,23 @@ new Vue({
                 ? true
                 : false
         },
-        showErrorLabelMaterials: function (product) {
+        showErrorLabelMaterials: function (product)
+        {
             // no materials
-            return (this.isEmptyResource(product['materials']))
-                ? true
-                : false
+            const re = /^CF-ST*/g
+            if (this.isEmptyResource(product['materials']))
+            {
+                if (((product['modelCode'] || '').match(re) || []).length)
+                {
+                    if (product['rejected_materials'])
```

Comment:
this method returns a bool, depending if we want to show a label or not. You don't want to set it dirty here. If you need to, please do it in the proper place, maybe something like:
```
if(!this.showErrorLabelMaterial(product) && product['rejected_materials']){
	product.dirty=true
}
```